internal/isoeditor/rhcos.go

```diff
@@ -18,10 +19,21 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-//go:generate mockgen -package=isoeditor -destination=mock_editor.go . Editor
+const rootfsServiceConfigFormat = `[Service]
+Environment=http_proxy={{.HTTP_PROXY}}
+Environment=https_proxy={{.HTTPS_PROXY}}
+Environment=no_proxy={{.NO_PROXY}}`
```

Comment:
When we added the variables to the installed agent back then, we added the proxy env vars in both capitalization since there is no standard for it.

Consider adding to the service the following (although might not be needed for curl - but have you covered if that changes)
```
Environment=HTTP_PROXY={{.HTTP_PROXY}}
Environment=HTTPS_PROXY={{.HTTPS_PROXY}}
Environment=NO_PROXY={{.NO_PROXY}}
```

Reply:
sure, added in https://github.com/openshift/assisted-service/pull/1090